tuf/tuf.go

```diff
@@ -173,6 +173,11 @@ func (tr *Repo) RemoveBaseKeys(role string, keyIDs ...string) error {
 	return nil
 }
 
+// GetAllLoadedRoles returns a list of all role entries loaded in this TUF repo, could be empty
+func (tr *Repo) GetAllLoadedRoles() []*data.Role {
```

Comment:
Not in the 0.2 timeframe, it's a big change with a lot of caution needed, but yeah, I'd like to do it sooner rather than later.